MAIN_API.md

```diff
@@ -0,0 +1,398 @@
+# Overview
+
+The key distinction made by Hammock is between _method containers_ and _functions_. Method containers comprise classes and objects. Functions comprise methods and global functions. Mocking a method container allows the developer to mock the behavior of any method within it. Hence, the mocking interface for method containers is called a _method mock container_. When a method is mocked through a method mock container, it instantiates a _function mock_, which is what actually mocks the method's behavior. The function mock is the unit of mocking within Hammock, since Hammock ultimately mocks functions.
```

Comment:
Method containers comprise _of_ classes and objects?

Reply:
Idem for `Function comprise methods and global functions`.

Reply:
In my mind it's "are composed of" or "comprise" (based on dictionary.com).

Reply:
Lexidor English isn't always the best.

Reply:
TBH `comprise` always gets me too.